src/java.base/share/classes/java/lang/System.java

```diff
@@ -2018,14 +2019,19 @@ public static void runFinalization() {
      *             linked with the VM, or the library cannot be mapped to
      *             a native library image by the host system.
      * @throws     NullPointerException if {@code filename} is {@code null}
+     * @throws     IllegalCallerException If the caller is in a module that
+     *             does not have native access enabled.
```

Comment:
The exception description is fine, just noticed the other exception descriptions start with a lowercase "if", this one is different.

Reply:
I'll fix this. Note that in `ModuleLayer.Controller`, all `@throws` start with capital letter, which is probably where I copied/pasted this from. I'll fix all, except for `ModuleLayer` where, for consistency, I think upper case is better.